src/popup/components/Modals/AssetSelector.vue

```diff
@@ -86,15 +100,26 @@ export default defineComponent({
     const loading = ref(true);
     const searchTerm = ref('');
     const isFullyOpen = ref(false);
+    const pageNumber = ref(1);
+    const innerScrollElem = ref<HTMLElement>();
 
+    const { setScrollConf } = useScrollConfig();
+    const { initViewport } = useViewport();
     const { accountAssetsFiltered } = useAccountAssetsList({
       searchTerm,
       withBalanceOnly: props.withBalanceOnly,
     });
 
-    const accountAssetsToDisplay = computed(() => (props.protocol)
-      ? accountAssetsFiltered.value.filter(({ protocol }) => protocol === props.protocol)
-      : accountAssetsFiltered.value);
+    const appInnerElem = computed<HTMLElement | null | undefined>(
+      () => innerScrollElem.value?.parentElement?.parentElement,
```

Comment:
Which element exactly is accessed here?
To be honest using `parentElement` twice in the Vue environment looks like rather bad approach.